fission-core/library/Fission/AWS/Route53.hs

```diff
@@ -1,3 +1,25 @@
-module Fission.AWS.Route53 (module Fission.AWS.Route53.Class) where
+module Fission.AWS.Route53
+  ( module Fission.AWS.Route53.Class
+  , verifyFirstResource
+  , getValuesFromRecords
+  )
+where
 
-import Fission.AWS.Route53.Class
+import           Fission.AWS.Route53.Class
+
+import           Fission.Prelude
+import           RIO
+import           Network.AWS.Route53
+
+verifyFirstResource
+  :: ListResourceRecordSetsResponse -> Text -> Maybe ResourceRecordSet
+verifyFirstResource resp name = case headMaybe recordSets of
+  Nothing  -> Nothing
+  Just rrs -> if (rrs ^. rrsName) == (name <> ".") then Just rrs else Nothing
```

Comment:
Good news: I got stylish-haskell working
Bad news: it's a little overzealous some of the time

For instance, I'd like to split this `if` statement into multiple lines, but it always condenses it. Is this something I should fix locally or in the `.stylish-haskell.yaml`?

Reply:
@dholms which version of stylish-haskell are you running?

Reply:
0.10